src/map/packet_system.cpp

```diff
@@ -1018,18 +1018,30 @@ void SmallPacket0x028(map_session_data_t* const PSession, CCharEntity* const PCh
     int32 quantity  = data.ref<uint8>(0x04);
     uint8 container = data.ref<uint8>(0x08);
     uint8 slotID    = data.ref<uint8>(0x09);
+    CItem* PItem    = PChar->getStorage(container)->GetItem(slotID);
+    uint16 ItemID   = PItem->getID();
 
     if (container >= MAX_CONTAINER_ID)
     {
         ShowExploit(CL_YELLOW "SmallPacket0x028: Invalid container ID passed to packet %u by %s\n" CL_RESET, container, PChar->GetName());
         return;
     }
 
-    CItem* PItem = PChar->getStorage(container)->GetItem(slotID);
+    if (ItemID > 29311 && ItemID < 29340)
```

Comment:
I don't know if this check will have to happen anywhere else in C++land, but I'm sure we can codify this to be something a little less opaque.

I kind of dig:
```cpp
PItem->IsStorageSlip() -> bool
```